db/seedData.js

```diff
@@ -83,14 +79,15 @@ async function createInitialOrders() {
       {
         creatorId: 2,
         name: "Hungry Hippy",
-        subtotal: 21.10,
+        subtotal: 21.1,
       },
     ];
     const orders = await Promise.all(
       ordersToCreate.map((order) => createOrder(order))
     );
     console.log("Orders Created: ", orders);
     console.log("Finished creating orders");
+    return orders;
```

Comment:
I dont believe we need to return out of this function, since the data is getting console log'd and stored in the database

Reply:
Done.